mobile/src/src/components/Login/LoginPage.jsx

```diff
@@ -53,74 +61,108 @@ const LoginPage = ({ navigation }) => {
   };
 
   return (
-    <View style={styles.container}>
-      <Text style={styles.title}>Login Page!</Text>
-      <View>
-        <View style={styles.row}>
-          <Text>Email: </Text>
-          <TextInput
-            style={styles.textInput}
-            onChangeText={(text) => setEmail(text)}
+    <KeyboardAwareScrollView>
+      <View style={styles.container}>
+        <View>
+          <Image
+            style={styles.imageStyle}
+            source={require("../../../assets/templatedLogin.png")}
           />
         </View>
-        <View style={styles.row}>
-          <Text>Password: </Text>
-          <TextInput
-            style={styles.textInput}
-            onChangeText={(text) => setPassword(text)}
-          />
+        <View style={styles.inputBoxes}>
+          <View style={styles.row}>
+            <TextInput
+              style={styles.textInput}
+              label="uWaterloo Email"
+              onChangeText={(text) => setEmail(text)}
+            />
+          </View>
+          <View style={styles.row}>
+            <TextInput
+              style={styles.textInput}
+              label="Password..."
+              style={styles.textInput}
+              onChangeText={(text) => setPassword(text)}
+            />
+          </View>
         </View>
-      </View>
-      <View style={styles.buttonContainer}>
-        <Button title="Sign in" onPress={handleSignIn} />
-        <Button title="Sign up" onPress={signUpRedirect} />
-      </View>
-      <Modal visible={modalVisible} presentationStyle={"overFullScreen"}>
-        <View style={styles.container}>
-          <View style={styles.modalView}>
-            <Text style={{ alignItems: "center" }}>Error:</Text>
-            {errorMessage}
-            <Button
-              title="Close"
-              style={styles.buttonContainer}
-              onPress={() => setModalVisible(false)}
-            ></Button>
+        <View style={styles.buttonContainer}>
+          <Button
+            compact={true}
+            style={styles.buttonStyle}
+            mode="contained"
+            title="Sign in"
+            onPress={handleSignIn}
+          >
+            Login
+          </Button>
+          <View style={styles.divider}>
+            <Text>or</Text>
           </View>
+          <Button
+            compact={true}
+            style={styles.buttonStyle}
+            mode="outlined"
+            title="Sign up"
+            onPress={signUpRedirect}
+          >
+            Register
+          </Button>
         </View>
-      </Modal>
-    </View>
+        <Modal visible={modalVisible} presentationStyle={"overFullScreen"}>
+          <View style={styles.container}>
+            <View style={styles.modalView}>
+              <Text style={{ alignItems: "center" }}>Error:</Text>
```

Comment:
Let's add this style to the styles variable

Reply:
not sure what you meant, let me know if the change I have just made is what you meant. I did not touch any of the modal info, this is all what it was before.